cmd/tendermint/commands/reset.go

```diff
@@ -42,21 +47,37 @@ var ResetPrivValidatorCmd = &cobra.Command{
 	Use:     "unsafe-reset-priv-validator",
 	Aliases: []string{"unsafe_reset_priv_validator"},
 	Short:   "(unsafe) Reset this node's validator to genesis state",
-	Run:     resetPrivValidator,
 	PreRun:  deprecateSnakeCase,
+	RunE:    resetPrivValidator,
 }
 
 // XXX: this is totally unsafe.
 // it's only suitable for testnets.
-func resetAllCmd(cmd *cobra.Command, args []string) error {
-	return resetAll(config.DBDir(), config.P2P.AddrBookFile(), config.PrivValidatorKeyFile(),
-		config.PrivValidatorStateFile(), logger)
+func resetAllCmd(cmd *cobra.Command, args []string) (err error) {
+	config, err = ParseConfig()
```

Comment:
Is there some reason we're parsing the config at the top level in `ResetStateCmd` but in the helper for `ResetAllCmd` and `ResetPrivValidatorCmd`? Obviously it functions either way, but I think there's some maintenance benefit to being consistent about it.

Reply:
(and yes, I realize the helpers currently take the command instead of the config, but as they're unexported anyway we could switch that to a path)

Reply:
Not sure I follow. Both commands parse the config in their respective `RunE` handlers.

Reply:
Yeah, I was hoping to collapse some of that. But on second thought, given that this is a patch in an old version, maybe we don't gain that much from cleaning up the noise here vs. on master.